cdap-watchdog/src/main/java/co/cask/cdap/logging/serialize/LogSerializerUtil.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2014-2017 Cask Data, Inc.
```

Comment:
Oh sorry, Because this is a new class it should be `Copyright © 2017 Cask Data, Inc.`